sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/models/SearchRequest.java

```diff
@@ -208,6 +208,11 @@ public final class SearchRequest implements JsonSerializable<SearchRequest> {
      */
     private SearchQueryVector vector;
```

Comment:
Should this be deleted?

Reply:
Currently, `vector` is still part of the swagger. For now, since it's still part of the swagger but not publicly exposed, we haven't decided to make any manual changes to autorest-generated code.

Reply:
You could consider using a swagger transform to remove it like we did in .NET at https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/src/autorest.md#remove-vector-property if you want to clean this up.